src/print_event_cb.c

```diff
@@ -621,13 +627,17 @@ print_event_bin(void *cb_cookie, void *data, int size)
 {
 	(void) cb_cookie;
 
+	pthread_spin_lock(&OutputLock);
```

Comment:
This seems like a bad idea when we would like things to be quick.
Anything preventing us from using a single write? I'm still trying to track down where `data` and `size` are coming from.
Anyways something like this might work:
```
char data[size + sizeof(size)];
memcpy(data, &size, sizeof(size));
memcpy(data + sizeof(size), data, size);
write(fd, data, size(size) + size);
```